docs/pages/connect-your-client/gui-clients.mdx

```diff
@@ -10,9 +10,20 @@ work with Teleport.
 
 ### Prerequisites
 
-(!docs/pages/includes/edition-prereqs-tabs.mdx!)
+- A running Teleport cluster version (=teleport.version=) or above. If you want to get started with Teleport, [sign
```

Comment:
```suggestion
- A running Teleport cluster. If you want to get started with Teleport, [sign
```
To simplify this a bit and reduce the risk that a customer will think that using GUI clients is only available if you have the latest Teleport version.